eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -362,6 +364,7 @@ stages:
         container: ${{ variables.androidCrossContainer }}
         targetOS: android
         targetArchitecture: arm64
+        runTests: false
```

Comment:
False is already the default, no need to mention it.

Reply:
IMHO it bloats the YML. We will define new parameters soon, i.e. `buildPass` which will have a default value of `1` and we won't set that default in every job vertical.

Reply:
I agree with Viktor on this one. We don't run tests by default and running tests really only makes sense for full builds, not on verticals, so specifying `runTests: false` on all of these is a lot of bloat. To test the verticals we'll likely need to assemble an sdk+nugets and then run scenario tests against that which is either a separate stage or a separate pipeline.